src/OpenSage.Game/Logic/Object/Behaviors/SpawnBehavior.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace OpenSage.Logic.Object
 {
-    internal sealed class SpawnBehavior : BehaviorModule
+    internal sealed class SpawnBehavior : BehaviorModule, IUpdateModule
```

Comment:
This module doesn't appear to inherit from `UpdateModule`, either based on C&C3 XSDs or its `Persist` method. But it does need to be updated, so I've added this `IUpdateModule` to allow non-`UpdateModule` classes to participate in updating, without involving _every_ `BehaviorModule` class.